search/index.json

```diff
@@ -1901,7 +1901,7 @@
     "keywords": ["","fame","hall","of","student"],
     "tags": null,
     "summary": "The following students have completed projects which have significantly improved ZAP.\nTo qualify for this list students must complete a non trivial project and have their work either included in the ZAP core or in an add-on that has been published on the Marketplace. Projects that are not published by the time the students have finished can still qualify to be included if someone else completes their work and publishes it for them.",
-    "content": "following students have completed projects which significantly improved zap qualify list must complete non trivial project their work either included core addon that has been published marketplace not by time finished can still someone else completes publishes them you would like part student then just post developer group contact title date program links manos kirtas scanning websockets phase 2019 summer gsoc blog wiki david scrobonia hud 20162018 direct repo video kajan mohanagandhirasa authentication helper 2018 ryan wehe christopher laguna rian franey form handling 2016 wnter mwos sanchit kapoor bug tracker support aabha biyani dom xss scanner 2015 sebastian schulze advanced fuzzing 2014 alberto verza soap cosmin stefan access control testing rauf butt birt reporting 2013 alessandro secco zest enhancements enhanced http session pulasthi mahawithana saml 20 abdelhadi azouni cms robert koch websocket 2012 masters thesis guifre ruiz ajax spider crawler redesign codes: direct: directly mentored team gsoc: google code mwos: mozilla winter security "
+    "content": "following students have completed projects which significantly improved zap qualify list must complete non trivial project their work either included core addon that has been published marketplace not by time finished can still someone else completes publishes them you would like part student then just post developer group contact title date program links manos kirtas scanning websockets phase 2019 summer gsoc blog wiki david scrobonia hud 20162018 direct repo video kajan mohanagandhirasa authentication helper 2018 ryan wehe christopher laguna rian franey form handling 2016 winter mwos sanchit kapoor bug tracker support aabha biyani dom xss scanner 2015 sebastian schulze advanced fuzzing 2014 alberto verza soap cosmin stefan access control testing rauf butt birt reporting 2013 alessandro secco zest enhancements enhanced http session pulasthi mahawithana saml 20 abdelhadi azouni cms robert koch websocket 2012 masters thesis guifre ruiz ajax spider crawler redesign codes: direct: directly mentored team gsoc: google code mwos: mozilla security "
```

Comment:
The `\nTo` got chopped out?

Reply:
Uh actually it seems to be more than that, the content string doesn't make sense...maube it doesn't need to for search?

Reply:
I believe so, the content is processed: https://lunrjs.com/guides/core_concepts.html